### DISCUSSION TITLE
v0.5.2 Token Generation Slower Than v0.4.8

### DISCUSSION THREAD

**Discussion by yanghan-cyber** [open-webui/open-webui#8161] [General]
### Environment Details
- **Version**: v0.5.2
- **Deployment Method**: Docker
- **Backend**: Open-WebUI
- **API Management**: Self-hosted OneAPI for OpenAI API

### Issue Description
I've noticed a significant performance degradation in token generation speed when comparing version v0.5.2 to v0.4.8.

### Steps to Reproduce
1. Deploy Open-WebUI using Docker
2. Use OneAPI to manage OpenAI API
3. Compare token generation speed between v0.4.8 and v0.5.2

### Observed Behavior
- In v0.4.8: Token generation was faster and more responsive
- In v0.5.2: Token generation speed has noticeably decreased

### Expected Behavior
Token generation speed should remain consistent or improve between versions.

### Additional Information
- Can provide more detailed performance metrics if needed
- Willing to assist in debugging or providing more information

### Suggested Next Steps
- Investigate potential performance regressions
- Review changes between v0.4.8 and v0.5.2 that might impact token generation speed

**Comment by tjbck** [comment]
#8088
